.gitignore

```diff
@@ -14,3 +14,5 @@ vendor/
 .env
 
 build/
+
+.idea
```

Comment:
👉👈I know you already know about it, buuuuuut.... 

🔥https://pro.codely.tv/library/git-introduccion-y-trabajo-en-equipo/93728/path/step/59548217/ 🔥